images/ceph/commit_hash

```diff
@@ -1 +1 @@
-cec053b62872f1342c8ebd8bc6d5e9c21aa82a1e
\ No newline at end of file
+3e7492b9ada8bdc9a5cd0feafd42fbca27f9c38e
```

Comment:
in the past you've rebased your change to "remove boost submodule" on top of the latest commits from upstream ceph.  is that no longer needed?

Reply:
ahh I missed that. fixed. its not strictly needed but does improve the build speed.